.github/workflows/ci.yml

```diff
@@ -6,12 +6,12 @@ on:
 
 jobs:
   PHP_Lowest:
-    name: PHP wth lowest dependencies
+    name: PHP ${{ matrix.php-versions }} wth lowest dependencies
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.2']
+        php-versions: ['7.4', '8.0']
```

Comment:
I do not added lowest dependencies for PHP 8.1 yet. This won't work as it pulls in a buggy version of PHPUnit - one more reason to use a separate composer.json for the tools ^^ -.